src/components/Text.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { palette } from "../theme/palette";
+
+export const H2 = styled.h2`
+  color: ${palette.neutralDarker}  
+  size: 36px;
```

Comment:
I think if we use rems it'll be easier to integrate. And should this be `font-size`?

Reply:
lol whoops, I started with a set of style variants and then scrapped them and hastily pulled them into the css. thanks for catching.